### PR TITLE
Blog: Article about 3rd stage Graphical Bootloader utilizing OTA partitions for multiple apps

### PR DESCRIPTION
**Pull request by georgik**

### CODE REVIEW

**Review by f-hollow — APPROVED** [review]
Another great article!

Left some small suggestions.

Also, do you plan to add a cover picture? Alternatively, we can come up with a picture featuring John Lee.

File: content/blog/esp32-storing-and-switching-between-multiple-firmwares/index.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+---
+title: "ESP32: Store and Switch Between Multiple Firmwares from Flash Memory"
```
[the remark this answers is not in the document]
Comment (georgik) [reply content/blog/esp32-storing-and-switching-between-multiple-firmwares/index.md]: Good point. Changed to: How to switch between multiple ESP32 firmware binaries stored in the flash memory

File: content/blog/esp32-storing-and-switching-between-multiple-firmwares/index.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+ota_4,       app,  ota_4,     ,         2816K,
+```
+
+This partition table should be defined in your project and the following option must be enabled in your [sdkconfig](https://github.com/georgik/esp32-graphical-bootloader/blob/main/sdkconfig.defaults) file:
+
+```shell
+CONFIG_PARTITION_TABLE_CUSTOM=y
+```
+
+## Setting Up the Graphical Bootloader
```
[the remark this answers is not in the document]
Comment (georgik) [reply content/blog/esp32-storing-and-switching-between-multiple-firmwares/index.md]: Nice. Applied.

**Comment by urish** [comment]
Cool to see the graphical bootloader live on wokwi

**Comment by f-hollow** [comment]
I asked our designer to create a cover picture for this article featuring John Lee. They will try to finish it this week.

Anyway, it shouldn't be a blocker for the release of this article. We can always use a placeholder picture and merge this PR whenever we need to, then update the picture later.

**Comment by georgik** [comment]
@f-hollow base don the feedback, I've added step-by-step instructions for creating custom app for the bootloader, so that a developer can create the application from the scratch.

**Review by f-hollow — APPROVED** [review]
Thank you for adding step-by-step instructions on how to create a custom app for the bootloader!

Just a couple of nitpicks from me.

**Review by pedrominatel — APPROVED** [review]
LGTM

**Comment by f-hollow** [comment]
@pedrominatel Thank you!

@georgik As we agreed, I will merge the branch. I am not sure if I should do anything with one [comment](https://github.com/espressif/developer-portal/pull/63#discussion_r1691053303), I left it unresolved. Feel free to review it.

**Pull request merged**